rest/tests/test_rest.py

```diff
@@ -765,11 +765,11 @@ def test_missing_learning_resource(self):
         self.get_learning_resource(repo_slug2, lr2_id)
 
     def test_filefield_serialization(self):
```

Comment:
Can you add an assertion somewhere (maybe in `test_rest_authorization.py` in `test_static_assets_get()`) which asserts the new `name` field?